sample/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,7 @@
 
     <string name="hello_world">Hello world!</string>
     <string name="action_settings">Settings</string>
+
+    <string name="mapboxAccessToken"></string>
```

Comment:
could you add an existing ID and token so that users can see this in the sample app without having to create their own?